### DISCUSSION TITLE
Upgrade Script Raspberry Pi releases

### DISCUSSION THREAD

**Discussion by GammaG** [DNSCrypt/dnscrypt-proxy#1906] [Show and tell]
Hey all,

I use dnscrypt-proxy as my default resolver in the internal network on a raspberry pi.
I've created an upgrade script if anyone is interested.
https://github.com/GammaG/DNSCryptProxy-Upgrade/blob/master/upgradeProxy.sh 
Requires the version number that should be upgraded to argument and sudo permissions.

Best Regards

**Comment by jedisct1** [comment]
This is great, but why not use the script from the documentation? https://github.com/DNSCrypt/dnscrypt-proxy/wiki/Updates#configuration
